documentation/source/development/engine-design/dependencies.rst

```diff
@@ -0,0 +1,30 @@
+Dependency management
+=====================
+
+- In general we try to keep the number of dependencies at a minimum.
+- We avoid to add dependencies directly to the project repository because they increase the size of the repository and we have to update them manually.
+
+- Instead, we prefer to use `conan package manager <https://conan.io/>`__ which allows us to get most dependencies from `conan center <https://conan.io/center/>`__.
+
+Conan package manager
+---------------------
+
+- The list of currently used dependencies can be found in `conanfile.py <https://github.com/inexorgame/vulkan-renderer/blob/master/conanfile.py>`__ in the root folder of the repository.
```

Comment:
Link might break in old documentations if we move that file. Better use a relative link to that file directly, but it is not supported by sphinx https://github.com/sphinx-doc/sphinx/issues/701

Reply:
We could also just mention `conanfile.py`?
So people could look it up manually.

Reply:
Done.